Makefile

```diff
@@ -34,26 +30,28 @@ CONAN_OPTIONS   :=
 .DEFAULT: help
 .SILENT: help
 help::
-	echo "Usage: make <target>"
+	$(call print_help_usage)
 	echo
 
+# Setup targets ---------------------------------------------------------------
 include Makefile.setup
-include Makefile.all
 
 # Workspace targets -----------------------------------------------------------
 .PHONY: lockfile status deploy sphinx doxygen docker-all docker-test docker-release purge-all export-cli smoketest smoketest-deps
 help::
-	echo "Available workspace targets:"
-	echo "  smoketest       to run BATS system tests"
-	echo "  sphinx          to generate Sphinx documentation"
-	echo "  doxygen         to generate Doxygen documentation"
-	echo "  deploy          to deploy Cloe to INSTALL_DIR [=${INSTALL_DIR}]"
-	echo "  deploy-cli      to install cloe-launch with pip"
-	echo "  export-cli      to export cloe-launch-profile conan recipe"
+	$(call print_help_section, "Available workspace targets")
+	$(call print_help_target, status, "show status of each of the Conan packages")
```

Comment:
Was it intended to list `status` both as workspace and as development target?

Reply:
Oops. Good catch!